server/channels/app/user_test.go

```diff
@@ -1095,7 +1129,7 @@ func TestPermanentDeleteUser(t *testing.T) {
 	err1 = sqlStore.GetMasterX().Select(&bots2, "SELECT * FROM Bots")
 	assert.NoError(t, err1)
 	assert.Equal(t, 0, len(bots2))
-
+	
```

Comment:
Unrelated change?

Reply:
Not sure why this changed - I can't seem to see the change here should I take the blank line out?